ape_ens/converter.py

```diff
@@ -16,28 +16,27 @@ class ENSConversions(ConverterAPI):
     @cached_property
     def mainnet_provider(self) -> Optional[Web3Provider]:
         provider = self.network_manager.active_provider
-        if (
```

Comment:
The changes in this file are only a refactor (invert if statement) and I added a NOTE about what is going on.
Otherwise there are not logic changes.